guides/rails_guides/generator.rb

```diff
@@ -146,14 +139,21 @@ def generate?(source_file, output_file)
       def generate_guide(guide, output_file)
         output_path = output_path_for(output_file)
         puts "Generating #{guide} as #{output_file}"
-        layout = @kindle ? "kindle/layout" : "layout"
+        layout = @epub ? "epub/layout" : "layout"
 
         view = ActionView::Base.with_empty_template_cache.with_view_paths(
           [@source_dir],
           edge:     @edge,
           version:  @version,
+<<<<<<< HEAD
```

Comment:
I don't think this should be here.

Reply:
Thanks! Fixed in 6f2f0a7e7103e23abe082c07c430ddda425f5e97

Reply:
Ah, sorry, and thanks!